server/routes/api/student.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const router = express.Router();
+var { Course, SeatingLayout } = require('../../dbSchemas/attendanceSchema');
+
+
+// allows updating a seating layout given a student id
+router.post('/updateStudentCourseView', (req, res) => {
+    let seatingLayoutID = req.body.seatingLayoutID;
+    let layout = req.body.layout;
```

Comment:
We should also be able to update the name, capacity, and dimensions of the seating layout.